en/patterns/tree-grid.md

```diff
@@ -1,47 +1,48 @@
+The Tree Grid Pattern can be used as an alternative starting point when designing a tabular data layout with a parent-child relationship between the records. It provides a full-blown contentwise and stylewise Tree Grid Component, on which all the Grid functionality provided in the Components Library can be applied.
 
 <img class="responsive-img" src="../images/tree_grid.png" srcset="../images/tree_grid@2x.png 2x" />
-
-List of all the Grid functionality:
-- Sorting
-- Filtering
-- Summaries
-- Paging
-- Editing
-- Column Moving
-- Column Resizing
-- Column Hiding
-- Display Density
-- Excel Style Filtering
-- Advanced Filtering
-- GroupBy
-- Pinning
-- Row Selection
-- Row Editing
-- Active Cell
-- Export
-- Toolbar
-
-All Grid features are applicable to the Tree Grid Pattern.
-
+<div class="divider--half"></div>
+
+A list of all the Grid functionalities can be found on the [Grid](../components/grid.md) component page, and they are all applicable to the Tree Grid Pattern. The applicable features are the following:
+
+- [Grid Toolbar](../components/grid-toolbar.md)
+- [Grid Export](../components/grid-export.md)
+- [Grid Group By](../components/grid-grouping.md)
+- [Grid Column Pinning](../components/grid-column-pinning.md)
+- [Grid Column Hiding](../components/grid-column-hiding.md)
+- [Grid Column Moving](../components/grid-column-moving.md)
+- [Grid Column Resizing](../components/grid-column-resizing.md)
+- [Grid Sorting](../components/grid-sorting.md)
+- [Grid Row Filter](../components/grid-row-filter.md)
+- [Grid Excel Style Filtering](../components/grid-excel-style-filter.md)
+- [Grid Advanced Filtering](../components/grid-advanced-filter.md)
+- [Grid Row Selection](../components/grid-row-selection.md)
+- [Grid Editing](../components/grid-editing.md)
+- [Grid Display Density](../components/grid-display-density.md)
+- [Grid Paging](../components/grid-paging.md)
+- [Grid Summaries](../components/grid-summaries.md)
+- [Grid Skeleton](../components/grid-skeleton.md)
 
 > [!WARNING]
-> In Sketch, after inserting the Tree Grid Pattern, you can trigger `Detach from Symbol` only if you need to alter the number of cells provided, in order to show all the dimensions of your data and insert as many visible records as you want. The individual Components and Features, however, must stay intact and not be detached!
+> In **Figma**, after inserting any of the Lists Patterns, in case you need more items than provided, you need to detach it from the `Detach Component` option after right-clicking on the component. After detaching, you will be able to increase the columns, and you will need to adjust their position manually. We highly recommend not detaching any other components nested in the Tree Grid because this might cause problems when generating the components in the App Builder.
```

Comment:
Lists Patterns?

Reply:
Tree Grid Pattern ... I've had problems and errors with the commits, and the previous one is the correct one (there I'm talking about the Tree Grid) ... something messed up here. I will fix it.

Reply:
Fixed